lower-constraints.txt

```diff
@@ -0,0 +1,93 @@
+alabaster==0.7.10
```

Comment:
nit: This file is referenced anywhere is it? Does not harm though...

Reply:
yeah, probably not used I guess.